### DISCUSSION TITLE
svg baseProfile

### DISCUSSION THREAD

**Discussion by sugizo** [niklasf/python-chess#992] [Ideas]
nice to lease svg by it's own syntax, no tiny any more

**ref**
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/baseProfile

thanks and best regards

**Comment by niklasf** [comment]
Thanks for the heads-up. Removed the deprecated attributes via bd371f68b631499d9107c20f1f785b111c547f62.
